app/models/hearings/forms/base_hearing_update_form.rb

```diff
@@ -341,6 +359,32 @@ def change_type
     end
   end
 
+  def determine_create_email_type
+    if only_time_updated_or_timezone_updated?
+      "updated_time_confirmation"
+    elsif conversion_to_virtual?
+      "convert_to_virtual_confirmation"
+    else
+      "confirmation"
+    end
+  end
+
+  def determine_delete_email_type
+    if conversion_to_not_virtual?
+      "convert_to_virtual_confirmation"
+    else
+      "cancellation"
+    end
+  end
+
+  def determine_email_type(action)
+    if action == "create_virtual_hearing"
+      determine_create_email_type
+    elsif action == "delete_virtual_hearing"
+      determine_delete_email_type
```

Comment:
per my comment above, this could just return 
```suggestion
      "convert_to_not_virtual_confirmation"
```
and we could remove `determine_delete_email_type` method

Reply:
I removed that method and returned just the result of the cancelled method as above.